app/test/integration_test.go

```diff
@@ -206,71 +185,6 @@ func (s *IntegrationTestSuite) TestMaxBlockSize() {
 	}
 }
 
-func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
```

Comment:
this test seems redundant with TestSubmitPayForBlob [here](https://github.com/celestiaorg/celestia-app/blob/be3acf0a79849af0dafda5eecb0644cfc08c58b0/pkg/user/signer_test.go#L48-L58)

Reply:
originally this was to test for the canonical api, so removing makes sense. we also don't need to be testing user.TxOptions here